test/e2e/create_cluster_test.go

```diff
@@ -1448,7 +1447,6 @@ func testSwitchFromPublicToPrivate(ctx context.Context, client crclient.Client,
 			obj.Spec.Platform.AWS.EndpointAccess = hyperv1.Private
 		})
 		g.Expect(err).ToNot(HaveOccurred(), "failed to update hostedcluster EndpointAccess")
-
-		e2eutil.ValidatePrivateCluster(t, ctx, client, hostedCluster, clusterOpts)
```

Comment:
why are we losing this signal?

Reply:
This was not actually testing anything as the cluster is validated before the switch (all COs ready, CV rolled out, nodes ready).  Then we switch and immediately revalidate, all assertions pass in 0s.

What we should really do is check the KAS and oauth routes and make sure that they transition properly.